.circleci/config.yml

```diff
@@ -9,18 +9,30 @@ orbs:
 workflows:
   version: 2
 
+  terraform_plan_apply:
```

Comment:
This should only run on `main` branch. Currently runs on any branch and pull requests